src/node-fallbacks/url.js

```diff
@@ -741,10 +741,14 @@ export { Url as Url };
 
 export var pathToFileURL;
 export var fileURLToPath;
+export var domainToASCII;
+export var domainToUnicode;
 
 if (lazy) {
   pathToFileURL = lazy("pathToFileURL");
   fileURLToPath = lazy("fileURLToPath");
+  domainToASCII = lazy("domainToASCII");
```

Comment:
lol i don't think this file is loaded anymore

Reply:
@Jarred-Sumner Do you mean the fallback? How does lazy loading Bun bits work in the bundled/browserified scenario?

Reply:
Originally, we shared this code between runtime and client-side bundles. This `lazy` came from that. It isn't actually loaded in browser builds. This is dead code.